src/smart-components/group/group.js

```diff
@@ -63,10 +65,21 @@ const Group = ({
   const location = useLocation();
 
   useEffect(() => {
-    fetchData(uuid);
-    insights.chrome.appObjectId(uuid);
-    return () => insights.chrome.appObjectId(undefined);
-  }, []);
+    fetchSystemGroup();
+    if (uuid !== 'default-access') {
+      fetchData(uuid);
+      insights.chrome.appObjectId(uuid);
+      return () => insights.chrome.appObjectId(undefined);
+    } else {
+      if (systemGroupUuid) {
+        fetchData(systemGroupUuid);
+        insights.chrome.appObjectId(systemGroupUuid);
+        return () => insights.chrome.appObjectId(undefined);
+      } else {
+        fetchSystemGroup();
+      }
```

Comment:
I think that this could be simplified
```suggestion
    const currUuid = uuid !== 'default-access' ? uuid : systemGroupUuid;
    fetchData(currUuid);
    if (currUuid) {
      insights.chrome.appObjectId(currUuid);
      return () => insights.chrome.appObjectId(undefined);
    }
```